examples/benchmark-memory.php

```diff
@@ -25,20 +25,20 @@
         }
     }
 }
-$r = (int) \round((\array_key_exists('r', $args) ? (int) $args['r'] : 2) * 1_000);
+$r = (int) \round((\array_key_exists('r', $args) ? (int) $args['r'] : 2));
 
 $runs = 0;
 
-if (5_000 < $t) {
+if (5 < $t) {
     EventLoop::delay($t, function () {
         EventLoop::getDriver()->stop();
     });
 }
 
-EventLoop::repeat(1, function () use (&$runs) {
+EventLoop::repeat(0.001, function () use (&$runs) {
     $runs++;
 
-    EventLoop::repeat(1_000, function (string $watcher) {
+    EventLoop::repeat(0.001, function (string $watcher) {
```

Comment:
This has been 1000ms so it should be 1s now.

Reply:
Thank you. I've fixed it to 1 second. Although, I didn't notice any difference in memory consumption or anything else.